bindings/java/CMakeLists.txt

```diff
@@ -141,9 +141,35 @@ target_include_directories(java_workloads PUBLIC ${JNI_INCLUDE_DIRS})
 
 set(CMAKE_JAVA_COMPILE_FLAGS "-source" "1.8" "-target" "1.8")
 set(CMAKE_JNI_TARGET TRUE)
-set(JAR_VERSION "${FDB_MAJOR}.${FDB_MINOR}.${FDB_REVISION}")
+
+# build a manifest file
+set(JAR_VERSION "${FDB_MAJOR}.${FDB_MINOR}.${FDB_PATCH}")
+string(TIMESTAMP BND_LAST_MODIFIED_SEC "%s")
+set(MANIFEST_TEXT "Manifest-Version: 1.0
+Bnd-LastModified: ${BND_LAST_MODIFIED_SEC}000
+Build-Jdk: ${Java_VERSION_STRING}
+Built-By: CMake ${CMAKE_VERSION}
+Bundle-Description: FoundationDB java binding and API
+Bundle-ManifestVersion: 2
+Bundle-Name: fdb-java
+Bundle-SymbolicName: com.foundationdb
+Bundle-Version: ${JAR_VERSION}
+Created-By: CMake ${CMAKE_VERSION}
+Name: com.foundationdb
+Implementation-Title: fdb-java
+Implementation-Version: ${CURRENT_GIT_VERSION}
+Signature-Version: ${JAR_VERSION}
```

Comment:
I was having some amount of trouble understanding this from the documentation on manifest files, but is this field relevant given that the jar isn't signed?

Reply:
Good catch, I think you're right.  I removed it.